src/frontend/src/pages/new-challenge/components/submit-popup.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
```

Comment:
```suggestion
import React, { useState } from "react";
```
This way, we can use useState instead of React.useState as in the React docs.

Reply:
Could you explain more regarding this matter?

Reply:
Importing the ```useState``` function means that you don't have to use ```React.useState``` later in the file. useState is not a variable that should be overridden, so it's fine to have it as a local variable.